README.md

```diff
@@ -1,19 +1,56 @@
 # Core banking
 
-This is a sample project showcasing a core banking application with microservices. It consists of the following services:
+Showcasing legacy system integration for a core banking application based on microservices. 
 
-- Data access services
-- Data service
-- Domain service
+## Introduction
 
-## Data access services
+This repository contains a sample core banking application architecture, demonstrating legacy database system integration using flexible, resilient and scalable components. The services used for data access are providing a standard query interface over several legacy data systems, unifying the method of access regardless of the original database technologies, storage format, physical location and configurational details involved.
 
-The following services are part of the data access services (`data-access-services`):
+The sample use-case built on this architecture is a retail banking web application used by back-office agents for credit rating of prospective clients. The application is using (semi)automated checks based on data supplied by applicants as well as pulling data from the legacy systems (e.g. CRM, transactions database).
 
-- Accounts service (`accounts-service`)
-- Transactions service (`transactions-service`)
+## Architectural overview
 
-The `data-access-services` sub-project is a multi-module Maven project which includes a library module (`lib`) for sharing configuration and code between the data access services.
+![core-banking-architecture](https://github.com/DeloitteDigitalAT/core-banking/blob/master/docs/core-banking-architecture.png)
```

Comment:
I'm not able to see the image, is it only me?
https://github.com/DeloitteDigitalAT/core-banking/tree/2340-upd-readme

Reply:
Not just you :) The link is referring to master branch `/blob/master/docs/...` so it will be only visible once this PR is merged. Sorry about that, I was not sure if it works with relative paths or not, so I picked the more explicit way of full URL. I can double-check the images after merge.